src/components/note-containers/note-containers.js

```diff
@@ -3,17 +3,37 @@ import './note-containers.css';
 import TodoData from "../../service/service";
 
 export default class NoteContainers extends Component {
+    
+    todoData = new TodoData();
 
-   
+    state = {
+        notes: []
+    };
 
+    componentDidMount() {
+        this.todoData
+        .getAllNotes()
+        .then((notes) => {
```

Comment:
can be done without then, but with async/await to make code a bit more readable 

look on first answer here
https://stackoverflow.com/questions/54495711/async-await-vs-then-which-is-the-best-for-performance